.credo.exs

```diff
@@ -19,7 +19,8 @@
         {Credo.Check.Readability.SinglePipe, []},
         {Credo.Check.Readability.Specs, []},
         {Credo.Check.Readability.StrictModuleLayout, []},
-        {Credo.Check.Readability.WithCustomTaggedTuple, []},
+        # Throwing an exception because we're using `with` as a variable name
+        # {Credo.Check.Readability.WithCustomTaggedTuple, []},
```

Comment:
I could go and open an issue and maybe see this fixed, but I have not the energy, at this moment, nor do I feel it important enough...